app/components/members/row_component.rb

```diff
@@ -89,13 +89,12 @@ def roles_label
       label
     end
 
-    def role_form_cell
-      Members::RoleFormCell.new(
+    def role_form_component
+      render Members::RoleFormComponent.new(
```

Comment:
nitpicking a bit: if it's returning the rendered component, it should not be named `role_form_component` but maybe `role_form`.